src/PrestaShopBundle/Resources/config/services/adapter/attribute_group.yml

```diff
@@ -2,6 +2,29 @@ services:
   _defaults:
     public: true
 
+  PrestaShop\PrestaShop\Adapter\AttributeGroup\CommandHandler\AddAttributeGroupHandler:
+    autowire: true
+    tags:
+      - name: tactician.handler
+        command: PrestaShop\PrestaShop\Core\Domain\AttributeGroup\Command\AddAttributeGroupCommand
+
+  PrestaShop\PrestaShop\Adapter\AttributeGroup\CommandHandler\EditAttributeGroupHandler:
+    autowire: true
+    tags:
+      - name: tactician.handler
+        command: PrestaShop\PrestaShop\Core\Domain\AttributeGroup\Command\EditAttributeGroupCommand
+
+  PrestaShop\PrestaShop\Adapter\AttributeGroup\QueryHandler\GetAttributeGroupForEditingHandler:
+    autowire: true
+    tags:
+      - name: tactician.handler
+        command: PrestaShop\PrestaShop\Core\Domain\AttributeGroup\Query\GetAttributeGroupForEditing
+
+  PrestaShop\PrestaShop\Adapter\Product\AttributeGroup\AttributeGroupViewDataProvider:
+    arguments:
+      - "@=service('prestashop.adapter.legacy.context').getContext().language.id"
+      - '@PrestaShop\PrestaShop\Core\ConfigurationInterface'
```

Comment:
I believe this one can be removed since you moved the services in another namespace in another PR no?

Reply:
Yes you are correct, removed it